docs/tutorials/fundamentals/tutorial-swapping-token.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Tutorial - Swapping UOS from Ultra to Ethereum'
```

Comment:
Please move it to a tutorials section and rename to `How to swap UOS from Ultra to Ethereum`